dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/localfiletreetable/Ds3GetJob.java

```diff
@@ -158,24 +157,38 @@ public void executeJob() throws Exception {
                         totalSent.addAndGet(l);
                     });
                     getJob.attachObjectCompletedListener(obj -> {
+                        final Calendar currentTime = Calendar.getInstance();
```

Comment:
The new java 8 time libraries should be used rather than the old pre java 8 time libraries.